test/support/serialization_testing.rb

```diff
@@ -10,4 +17,13 @@ def with_adapter(adapter)
   ensure
     ActiveModel::Serializer.config.adapter = old_adapter
   end
+  alias_method :with_configured_adapter, :with_adapter
```

Comment:
Before SerializableResource, the only way to specify an adapter was global, as an option to render, or by building the serializer and adapter yourself.. now we don't need global quite as much